tests/llm/test_anthropic/test_stream.py

```diff
@@ -30,26 +30,21 @@ def test_iterable_model(model, mode, stream, client):
         assert isinstance(m, UserExtract)
 
 
-@pytest.mark.parametrize("model, mode, stream", product(models, modes, [True, False]))
+@pytest.mark.parametrize("model, mode", product(models, modes))
 @pytest.mark.asyncio
-async def test_iterable_model_async(model, mode, stream, aclient):
+async def test_iterable_model_async(model, mode, aclient):
     aclient = instructor.from_anthropic(aclient, mode=mode)
-    model = await aclient.messages.create(
+    model = await aclient.chat.completions.create(
         model=model,
```

Comment:
Removed stream=True and false tests since I migrated the underlying call to use create_iterable so that we have a more consistent handling of iterables.